analytics_data_api/v0/documents.py

```diff
@@ -27,13 +28,13 @@ class RosterEntry(Document):
     """
 
     course_id = Keyword()
-    user_id = Integer()
+    user_id = Short()
```

Comment:
Sergey, I asked to think where to apply this `Short` format, and not just apply. Think about where this format good to use. What do you think can be on the course more than 32,767 students?

Reply:
Done. I think that only this field may be greater than 32,767